azure-pipelines.yml

```diff
@@ -150,19 +150,15 @@ jobs:
     vmImage: ubuntu-18.04
   steps:
   - template: ci/install-package-dependencies.yml
-  - bash: |
-      set -x
-      sudo util/get-toolchain.py \
-        --install-dir="$TOOLCHAIN_PATH" \
-        --release-version="$TOOLCHAIN_VERSION" \
-        --update
-    displayName: Install toolchain
   - bash: |
       . util/build_consts.sh
-      ./meson_init.sh -A
-      ninja -C "$OBJ_DIR" \
-        sw/device/otp_img/otp_img_export_fpga_cw310 \
-        sw/device/lib/testing/test_rom/test_rom_export_fpga_cw310
+      ci/bazelisk.sh build \
+      //hw/ip/otp_ctrl/data:test_rma_image \
+      //sw/device/lib/testing/test_rom:test_rom_fpga_cw310_scr_vmem
+      mkdir -p $BIN_DIR/sw/device/otp_img
+      mkdir -p $BIN_DIR/sw/device/lib/testing/test_rom
+      cp bazel-bin/hw/ip/otp_ctrl/data/test_rma_image.vmem $BIN_DIR/sw/device/otp_img/otp_img_fpga_cw310.vmem
+      cp $(find bazel-out/* -name test_rom_fpga_cw310.scr.39.vmem) $BIN_DIR/sw/device/lib/testing/test_rom/test_rom_fpga_cw310.scr.39.vmem
```

Comment:
Oof! Can you just put the path in? Otherwise, there's the possibility of weird error messages if there are ever 0 or 2 hits.

Reply:
I shouldn't use the whole path, I could reconstruct it but it's intentionally verbose to avoid collisions. I already have a cleanup task on it, but I can use a query instead of find.

Reply:
That sounds like a sensible idea. Happy to leave this for a follow-up.